classes/author/Repository.inc.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @file classes/author/Repository.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class author
+ *
+ * @brief A repository to find and manage authors.
+ */
+
+namespace APP\author;
+
+use APP\core\Request;
+use PKP\Services\PKPSchemaService;
+use Illuminate\Support\Facades\App;
+
+class Repository extends \PKP\author\Repository
+{
+    public function __construct(DAO $dao, Request $request, PKPSchemaService $schemaService)
```

Comment:
This doesn't add anything to the parent implementation, does it? If not, remove it.

Reply:
This one is necessary to correctly type hint the `DAO`. Since we use automatic dependency injection, the `\PKP\author\Repository` would inject the `\PKP\author\DAO` but we want the `\APP\author\DAO` to be injected here.